app/Integration/Article/Repository/InMemoryArticleRepository.php

```diff
@@ -63,4 +63,19 @@ public function add(Article $article): void
     {
         $this->articles->put((string) $article->id(), $article);
     }
+
+    public function looseConnection(): void
```

Comment:
THose doesn't make sense for me.

If you want to test an implementation failure, you can throw an exception as one of exposed in the throws doc blocks